pkg/epinio/promptRemove/services.vue

```diff
@@ -36,7 +36,11 @@ export default {
     },
 
     bounded() {
-      return this.value.filter(row => row.boundapps?.length);
+      return this.value?.reduce((acc, svc) => {
+        const apps = svc?.boundapps ?? [];
+
+        return [...acc, ...apps];
```

Comment:
we're not dealing with a lot of results, but `return acc.concat(apps);` is a bit neater (and faster)